info_terminal/info_terminal_gui/scripts/info_terminal.py

```diff
@@ -108,12 +110,17 @@ def GET(self):
 class Events(object):
     def GET(self):
         app.publish_feedback(Events.id)
+        blog = xmltodict.parse(requests.get(HENRY_BLOG_URL).text)
+        blog_events = []
+        for n in blog['rss']['channel']['item']:
+            blog_events.append(n["content:encoded"])
```

Comment:
I think it would be better to pull the content from `blog_events.append(n["description"])` as we won't have links to comments, etc. then.